src/rgw/rgw_crypt.cc

```diff
@@ -35,8 +35,6 @@ using namespace rgw;
 /**
  * Encryption in CTR mode. offset is used as IV for each block.
  */
-#warning "TODO: move this code to auth/Crypto for others to reuse."
```

Comment:
@joscollin the advantage if you revert the commit is we have the warning retained as comment and helps us trace back git histories better